br.com.eduk.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>br.com.eduk</id>
+  
+  <name>eduK</name>
+  <summary>A revolution in the social impact technology market, combining online education and income generation.</summary>
```

Comment:
Remove the article "A" in front and dot in the end.

Also the line doesn't describe what the app is for/does...

Reply:
Fix